pkgs/development/tools/java/jextract/default.nix

```diff
@@ -87,6 +92,7 @@ stdenv.mkDerivation {
     makeWrapper "${jdk20}/bin/java" "$out/bin/jextract" \
       --add-flags "--enable-preview" \
       --add-flags "--class-path $out/share/java/org.openjdk.jextract-unspecified.jar" \
+      --add-flags "-Djava.library.path=${llvmPackages.libclang.lib}/lib" \
```

Comment:
Please add a comment linking to the issue.

Reply:
Do you mean to this PR?

Reply:
To https://github.com/NixOS/nixpkgs/issues/247551

Reply:
Since we don't specify `-Djava.library.path` anymore, I think it's no need to link to that issue now.